src/refactor_nrepl/core.clj

```diff
@@ -184,7 +188,10 @@
 (defn find-in-project
   "Return the files in the project satisfying (pred ^File file)."
   [pred]
-  (-> find-in-dir (partial pred) (mapcat (dirs-on-classpath)) distinct))
+  (->> (dirs-on-classpath)
+       (pmap (partial find-in-dir pred))
```

Comment:
I chose `pmap` for simplicity, it's better than some tend to say

(because `pmap` won't create one thread per item... it takes the CPU count in account)

...it would be inefficient for very small inputs, but it's safe to say that most projects have multiple dirs in the classpath (especially in monorepos), plenty of namespaces, etc.

Reply:
Good choice! The small workloads that will get slightly slower will still be more than fast enough in the face of some fork/join overhead, I think  :)